### PR TITLE
chore: update docs regarding yarn v1

### PR DESCRIPTION
**Pull request by GermaVinsmoke**
Doc update for #2718

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -185,6 +185,8 @@ Also, have a look at [Frontend README.md](https://github.com/SigNoz/signoz/blob/
   yarn dev
   ```
 
+  **Note:** Please make sure to have classic version of yarn - `v1.22.19` installed in your system
```
Comment (palashgdev) [inline CONTRIBUTING.md:188]: can we have only one note

**Review by palashgdev — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (GermaVinsmoke) [reply CONTRIBUTING.md:188]: Okay, where should we keep it?

Comment (palashgdev) [reply CONTRIBUTING.md:188]: maybe at 188 line

**Review by palashgdev — DISMISSED** [review]

**Comment by GermaVinsmoke** [comment]
We can also think of creating a **Prerequisite** section which can be added above other two sections - with docker, without docker.
Prerequisite section can have details for other things also like, node version needs to be above `v16.15.0`

**Review by srikanthccv — DISMISSED** [review]
minor suggestion

**Review by palashgdev — APPROVED** [review]

**Comment by srikanthccv** [comment]
Not relevant anymore.

**Pull request closed without merging**